omnibus/config/software/libpcap.rb

```diff
@@ -19,30 +19,5 @@
 source url: "https://www.tcpdump.org/release/libpcap-#{version}.tar.xz"
```

Comment:
I would remove this to avoid fetching on omnibus side. This should also cut off some seconds from the build duration

Reply:
I can't yet. This is how we pick up the license. That is why I am so keen on the license stuff I've been doing lately. Once I can pick it up from bazel I can delete all the migrated omnibus files.